cohttp-lwt/src/s.ml

```diff
@@ -185,28 +187,19 @@ module type Client = sig
   type ctx
 
   (** Provide a function used to process requests.
-      Please see {!type:requester}.
+      Please see {!type:call}.
       The provided function is only used when no [ctx] argument is
       passed to the convenience functions below. *)
   val set_cache : call -> unit
 
-  (** processes a request.
-      Please see {!type:requester}.
-      @param ctx If provided, this is
-      {!val:Connection_cache.Make_no_cache.create}. *)
-  val request : ?ctx:ctx -> call
-
   (** [call ?ctx ?headers ?body ?chunked meth uri]
-      constructs a {!module:Request} using provided [headers],
-      [chunked], [meth] and [uri]. Then passes this {e request} and the
-      [body] to {!val:request}.
 
       @return [(response, response_body)] Consume [response_body] in a
-      timely fashion. Please see {!val:requester} about how and why.
+      timely fashion. Please see {!val:call} about how and why.
 
       @param chunked use chunked encoding if [true]. The default is
       [false] for compatibility reasons.
-      @param ctx If provided, {!val:request} is not used, but
+      @param ctx If provided, no connection cache is not used, but
```

Comment:
I am not sure about the wording of this parameters, I don't understand what you mean